manifests/mariadb101.pp

```diff
@@ -0,0 +1,31 @@
+# Class: yumrepos::mariadb101
+#
+# Installs the MariaDB 10.1 yum repository.
+#
+class yumrepos::mariadb101 (
+  $mariadb101_url = $yumrepos::params::mariadb101_url,
+  $mariadb101_enabled = $yumrepos::params::mariadb101_enabled,
+  $mariadb101_gpgcheck = $yumrepos::params::mariadb101_gpgcheck,
+  $mariadb101_includepkgs = $yumrepos::params::mariadb101_includepkgs,
+  $mariadb101_exclude = $yumrepos::params::mariadb101_exclude,
+) inherits yumrepos::params {
+
+  file { '/etc/pki/rpm-gpg/RPM-GPG-KEY-MariaDB':
```

Comment:
Since this is the same key as used by the MariaDB 10 class, please add a `! defined` check here (see `drush7.pp` for an example).

Reply:
ah, good call.